api_docs/changelog.md

```diff
@@ -30,6 +30,12 @@ format used by the Zulip server that they are interacting with.
 * `PATCH /realm`: Added `create_multiuse_invite_group` parameter
   to support setting the user group whose members can create invite link.
 
+* `GET /invites`: Now, the endpoint response for non-admin users includes
+   the multiuse invites created by them.
+
+* `DEL /invites/multiuse/{invite_id}`: Allow non-admins users to revoke
+   multiuse invites created by them.
+
```

Comment:
I am not sure whether we want a separate feature level entry for these API changes.